config/application.yml.example

```diff
@@ -70,6 +71,7 @@ production:
   otp_delivery_blocklist_findtime: '5'
   otp_delivery_blocklist_maxretry: '5'
   participate_in_dap: 'no'
+  password_pepper: 'rake secret'
```

Comment:
the file `config/application.yml` is used to manage keys not stored in git. That file is managed manually at present, though there is effort underway to automate. https://18f.slack.com/archives/identity-dev/p1473351874000106